### PR TITLE
refactor(link): refactor rendering api to template syntax

### PR DESCRIPTION
**Pull request by provydon**
## Pull request type

Jira Link: [INT-](url)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## What is the new behavior?

- 
- 
-

### CODE REVIEW

**Comment by lisilinhart** [comment]
Hi @provydon, thanks for the help. We'll take a look in the next days :)

File: src/components/Link/SbLink.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  <component
+    :is="as"
+    class="sb-link"
+    :class="classes"
+    :title="title"
+    :href="href"
+    :to="to"
+    v-on="$listeners"
+  >
+    <SbIcon v-if="icon" :size="'normal'" :name="icon"></SbIcon>
```
Comment (lisilinhart) [inline src/components/Link/SbLink.vue:11]: I think the `:size` can be removed

**Review by lisilinhart — CHANGES_REQUESTED** [review]

**Comment by lisilinhart** [comment]
@thiagosaife  I made some tests in the app and apart from the comment I made I think we can merge it, because it was working the same as before 
@provydon maybe you can check the comment :)

File: src/components/Link/SbLink.vue (hunk 1 above)
Comment (gustavomelki) [reply src/components/Link/SbLink.vue:11]: please, use self-close for components without content: https://v2.vuejs.org/v2/style-guide/?redirect=true#Self-closing-components-strongly-recommended

Comment (provydon) [reply src/components/Link/SbLink.vue]: @gustavomelki  i've self-closed the components without content, please review. Thanks

**Pull request closed without merging**